app/Services/ContestManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Gladiator\Services;
+
+use Gladiator\Repositories\UserRepositoryContract;
+
+class ContestManager
```

Comment:
hmm yea maybe a `Manager` service and then if we have more specific needs per Model we can break them out?